collectors/ebpf.plugin/README.md

```diff
@@ -178,35 +104,129 @@ process and thread creation.
-`ebpf.conf`. By default, the plugin uses the less expensive `entry` mode. You can learn more about how the
-plugin works using `entry` by reading this configuration file.
+Enable or disable the entire eBPF collector by editing `netdata.conf`.
+
+```bash
+cd /etc/netdata/   # Replace with your Netdata configuration directory, if not /etc/netdata/
+./edit-config netdata.conf
+```
 
-You can always edit this file with `edit-config`:
+You can also configure the eBPF collector's behavior by editing `ebpf.conf`.
 
 ```bash
-cd /etc/netdata/ # Replace with your Netdata configuration directory, if not /etc/netdata/
+cd /etc/netdata/   # Replace with your Netdata configuration directory, if not /etc/netdata/
 ./edit-config ebpf.conf
 ```
 
 ### `[global]`
 
-In this section we define variables applied to the whole collector and the other subsections.
+The `[global]` section defines settings for the whole eBPF collector.
 
 #### load
 
 The collector has two different eBPF programs. These programs monitor the same functions inside the kernel, but they
 monitor, process, and display different kinds of information.
 
 By default, this plugin uses the `entry` mode. Changing this mode can create significant overhead on your operating
-system, but also offer important information if you are developing or debugging software. The `load` option accepts the
+system, but also offer valuable information if you are developing or debugging software. The `load` option accepts the
 following values: ​
 
--   `entry`: This is the default mode. In this mode, Netdata monitors only calls for the functions described in the
-    sections above. When this mode is selected, Netdata does not show charts related to errors.
--   `return`: In this mode, Netdata also monitors the calls to function. In the `entry` mode, Netdata only traces kernel
-    functions, but with `return`, Netdata also monitors the return of each function. This mode creates more charts, but
-    also creates an overhead of roughly 110 nanosections for each function call.
+-   `entry`: This is the default mode. In this mode, the eBPF collector only monitors calls for the functions described
+    in the sections above, and does not show charts related to errors.
+-   `return`: In the `return` mode, the eBPF collector monitors the same kernel functions as `entry`, but also creates
+    new charts for the return of these functions, such as errors. Monitoring function returns can help in debugging
+    software, such as failing to close file descriptors or creating zombie processes.
+
+## Install and enable the eBPF collector
+
+Systems running _v1.22 or earlier_ of the Agent will need to install and enable the eBPF collector manually. The
+collector is enabled by default on later versions of the Agent.
+
+If you installed via the one-line installation script, 64-bit binary, or manually, you can append the `--enable-ebpf`
```

Comment:
the `--enable-ebpf` has been removed, please, remove the reference to it.

Reply:
Done!